### PR TITLE
HPCC-21405 ConfigMgr Advanced environment list disabled on navigating back

### PR DESCRIPTION
**Pull request by kenrowland**
Ensure Advanced radio button is un-checked on return to home screen of
ConfigMgr as checking the button enables the dropdown.

Signed-off-by: Ken Rowland <kenneth.rowland@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by kenrowland** [comment]
@miguelvazq Please review. 

Please note that there are a lot of whitespace changes due to removal of trailing blanks. The only change of interest is at the end where the 4 radio buttons are set to the un-checked state.

File: esp/files/configmgr.html [hunk 1]
```diff
@@ -267,140 +267,140 @@
...
-        *padding:1px; 
-        *margin:1px; 
-      } 
-     
-      #wizardTop .hd{    
-        padding:0px;   
+        color:white;
+        height:55px;
+        overflow:visible;
+        *padding:1px;
```
Comment (miguelvazq) [inline esp/files/configmgr.html]: Not sure what `*` does as a prefix but, I do see it throughout.

File: esp/files/configmgr.html [hunk 2]
```diff
@@ -267,140 +267,140 @@
...
+        height:30px;
+        padding:none;
+        border-bottom: 1px solid #808080;
         *border-bottom:none;
-       } 
-     
-     
-     .yui-dt td.classForSum 
+       }
+
```
Comment (miguelvazq) [inline esp/files/configmgr.html]: Minor: Extra spaces.

**Review by miguelvazq — COMMENTED** [review]
The Javascript looks fine.

**Review by miguelvazq — APPROVED** [review]

**Comment by miguelvazq** [comment]
Squash probably needed.

**Comment by kenrowland** [comment]
@richardkchapman Please merge

**Pull request merged**